tests/Containerfile

```diff
@@ -1,9 +1,7 @@
 FROM registry.gitlab.com/centos/automotive/sample-images/hirte/hirte-builder:1.0.0
-COPY ./builddir/bin/usr/local/bin/* /usr/local/bin
 
-CMD mkdir -p /usr/local/share/dbus-1/interfaces/
-COPY ./builddir/bin/usr/local/share/dbus-1/interfaces/* \
-     /usr/local/share/dbus-1/interfaces/
+COPY ./builddir/bin/usr/local/bin/* /usr/local/bin
```

Comment:
Changing this would conflict with https://github.com/containers/hirte/blob/28e1f6470b913eb19e72bad50e43ea612896b4b3/.github/workflows/e2e.yaml#L32

There has been discussion to redo the E2E CI testing in a different format anyhow, as right now everything is tightly coupled to GitHub Actions. Before changing CI I would like to wait for the outcome first.

Reply:
I think Dan meant to use make instead of meson in automation 
Each command here could be done with make command

Make PR introduced after e2e workflow
It means that hirte/.github/workflows/e2e.yaml
Will use make instead of meson, WDYT?

2nd option, create an issue

Reply:
Yes, I think we should be using Make commands rather then meson and random copies.

Reply:
Using Make primatives, allows all users to be able to contribut without forcing them to learn meson.

Reply:
One more question here, Dan
Current. build run into steps. 
1) pull and make/meson command with shared volume.
2) copy the binaries from shared volume to the new container.

Is it more efficient  to repeat step 1 without shared volume, run make install inside build container?
Apply  podman container commit, with compiled binaries in running container

Then step 2 could be reduces to a minimum commands?

Reply:
I would advise to go this approach, this way we could use the rpm generated by each workflow @raballew 
https://github.com/containers/hirte/issues/158, so I assume this PR can be closed, if you want to work on the new Issue, LMK